### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Noumanluckey8057**
Ai

### CODE REVIEW

**Comment by evandrofgarcia** [comment]
Amazing 👍

**Comment by V1e2n3k4a5t6e7s8w9a10r11l12u13** [comment]
Ok

**Comment by RaybenZ81** [comment]
😎🔥

**Comment by evandrofgarcia** [comment]
Wonderful

**Review by Abdulrahim011 — APPROVED** [review]
I need to put

**Review by Nataliedent621 — APPROVED** [review]

**Comment by evandrofgarcia** [comment]
Fantastic

File: README.md [hunk 1]
```diff
@@ -48,4 +48,31 @@ Translation files are here:
 https://github.com/pi-apps/pi-explorer/tree/master/src/languages
```
Comment (evandrofgarcia) [inline README.md:48]: README.md

**Review by Nataliedent621 — APPROVED** [review]

**Review by Handoyo1983 — APPROVED** [review]

**Review by EnnoYN — APPROVED** [review]

**Review by Noumanluckey8057 — COMMENTED** [review]
Om

**Review by Evandro775 — COMMENTED** [review]
Sure

**Comment by ahmedaswad** [comment]
!

**Review by Abdulrahim011 — APPROVED** [review]

**Comment by Dome79** [comment]
I want subscribes me front Now. Thanks domenico Loiacono Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by ahmedaswad** [comment]
!

بتاريخ الخميس، 16 مايو 2024، جاء من Dome79 ***@***.***>:

**Comment by Duracell1919** [comment]
Nice 👍

**Comment by ahmedaswad** [comment]
What's naec?

بتاريخ الخميس، 16 مايو 2024، جاء من Duracell1919 ***@***.***>:

**Comment by Noumanluckey8057** [comment]
Ok

**Comment by Noumanluckey8057** [comment]
Lo

**Comment by Noumanluckey8057** [comment]
Ok

**Review by Noumanluckey8057 — COMMENTED** [review]
Thnx[](url)

**Comment by Sutherlandjoseph** [comment]
Thanks

**Review by evandrofgarcia — COMMENTED** [review]
noumanluckey8057:patch-1

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Comment by evandrofgarcia** [comment]
Incredible

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by evandrofgarcia — COMMENTED** [review]
Obrigado meu amigo por me avisar!🙏🤜🏼🤛🏼👍

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Barinjan) [reply README.md:48]: +93784434308

Comment (Sutherlandjoseph) [reply README.md:48]: Hello

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Review by Sutherlandjoseph — APPROVED** [review]

**Comment by Williamdorcas** [comment]
Hello. How are you?

**Comment by RoarTammy72** [comment]
Why was 36 of my pie stolen out of my wallet without my consent by you

**Comment by 0912470398** [comment]
What

**Pull request closed without merging**